src/main/java/com/google/sps/servlets/Logout.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// Delete the session cookie and sign out user
+@WebServlet("/sign-out")
+public class Logout extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final int maxAge = 0;
+    Cookie newCookie = new Cookie("session", " ");
```

Comment:
It would be better if the session cookie was set to an empty string (no space). This may help prevent issues in the future.